544: Add traditional GP highlight color to the current sort column.

toolstack opened the pull request:
Resolves #544.

toolstack committed:
Add traditional GP highlight color to the current sort column.

ocean90 commented:
As noted in #544, the blue color should only be there if a user changes the sort order.

toolstack committed:
Add cell boarders back.

toolstack committed:
Sort the locales list by english name.

We can't trust that the list is in any kind of order, so just like when we return a partial list, order it by the english name.

toolstack committed:
Format the locales table to be as close to the final output as possible.

This avoid most "flicker" in Chrome, however there is still a "bounce" as Chrome looks to be miscalculating the column width during the first render of the table and then correcting it afterwards.

toolstack committed:
Update the table format in the projects list and set correct initial sort.

toolstack committed:
Flag the correct column as the default sort.

toolstack committed:
Coding standards cleanups.

toolstack committed:
Coding standards cleanups.

ocean90 commented on gp-templates/project.php:
```diff
@@ -44,23 +44,35 @@
+			<tr class="tablesorter-headerRow">
+				<th class="header tablesorter-header tablesorter-headerUnSorted"><?php _e( 'Locale', 'glotpress' ); ?></th>
+				<th class="header tablesorter-header tablesorter-headerUnSorted"><?php _ex( '%', 'locale translation percent header', 'glotpress' ); ?></th>
+				<th class="header tablesorter-header tablesorter-headerAsc"><?php _e( 'Translated', 'glotpress' ); ?></th>
```
The sorting is descending, needs to be `tablesorter-headerDesc`.

toolstack replied:
Fixed.

ocean90 commented on gp-templates/locales.php:
```diff
@@ -12,21 +12,32 @@
+				}
+
+				$i++;
+?>
```
This can be simplified to something like

``` php
$class = '';
foreach ( … ) {
    $class = ( 'odd' == $class ) ? 'even' : 'odd';
}
```

But: we don't need any PHP here. We can use the `:nth-child(odd)` selector in CSS instead.

toolstack replied:
Updated.

I agree we can do it with CSS, but I changed as little as possible in the update as I figured we'd do that through the redesign.

ocean90 reviewed (changes requested):
With the additional changes I don't have any flickering anymore.

toolstack committed:
Fixed default sort order class.

toolstack committed:
Simplified odd/even logic.

toolstack committed:
Coding standards cleanup.

toolstack committed:
Covert the sort order indicators from black to white.

toolstack committed:
Update highlight color for current sort column.
